sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ParDoTest.java

```diff
@@ -2018,6 +2018,34 @@ public void onTimer(OnTimerContext context) {
 
   @Test
   @Category({ValidatesRunner.class, UsesTimersInParDo.class})
+  public void testEventTimeTimerAlignBounded() throws Exception {
+    final String timerId = "foo";
+
+    DoFn<KV<String, Integer>, Integer> fn =
+        new DoFn<KV<String, Integer>, Integer>() {
+
+          @TimerId(timerId)
+          private final TimerSpec spec = TimerSpecs.timer(TimeDomain.EVENT_TIME);
+
+          @ProcessElement
+          public void processElement(ProcessContext context, @TimerId(timerId) Timer timer) {
+            timer.setForNowAlign(Duration.standardSeconds(1), Duration.millis(1));
```

Comment:
When I read this invocation, it is actually hard to tell which is the offset and which is the alignment. I wonder if there is a clever way to change the API so it could indicate them separately.